src/net/sf/memoranda/ui/htmleditor/HTMLEditor.java

```diff
@@ -245,8 +245,9 @@ public void actionPerformed(ActionEvent e) {
 					0,
 					0,
 					HTML.Tag.HR);
-			} catch (Exception ex) {
-				ex.printStackTrace();

```

Comment:
This can't be changed to parameter 'e' because it is inside an action handler method that is already using e as the parameter name.